nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/src/main/java/org/apache/nifi/processors/azure/storage/utils/AzureStorageUtils.java

```diff
@@ -54,6 +70,33 @@ public final class AzureStorageUtils {
             .defaultValue(AzureStorageCredentialsType.SAS_TOKEN)
             .build();
 
+    public static final PropertyDescriptor FILESYSTEM = new PropertyDescriptor.Builder()
+            .name("filesystem-name").displayName("Filesystem Name")
+            .description("Name of the Azure Storage File System (also called Container). It is assumed to be already existing.")
+            .addValidator(StandardValidators.NON_BLANK_VALIDATOR)
+            .expressionLanguageSupported(ExpressionLanguageScope.FLOWFILE_ATTRIBUTES)
+            .required(true)
+            .build();
+
+    public static final PropertyDescriptor DIRECTORY = new PropertyDescriptor.Builder()
+            .name("directory-name")
+            .displayName("Directory Name")
+            .description("Name of the Azure Storage Directory. The Directory Name cannot contain a leading '/'. The root directory can be designated by the empty string value. " +
+                    "In case of the PutAzureDataLakeStorage processor, the directory will be created if not already existing.")
+            .addValidator(new AbstractAzureDataLakeStorageProcessor.DirectoryValidator())
```

Comment:
I think `DirectoryValidator` should be moved here too or into an outer class. The util should not reference back the abstract class.